history-server.md

```diff
@@ -0,0 +1,76 @@
+# Spark History Server
+
+### Prerequisites
+
+Required software:
+* K8s cluster
+* [KUDO CLI Plugin](https://kudo.dev/docs/#install-kudo-cli) 0.7.3 or higher
+
+### Installing Spark Operator with History Server Enabled
+
+```
+kubectl kudo install ./kudo-operator \
+    -n <NAMESPACE> \
+    -p operatorImageName=mesosphere/kudo-spark-operator \
+    -p operatorVersion=latest \
```

Comment:
let's remove `-p operatorImageName=mesosphere/kudo-spark-operator` and `-p operatorVersion=latest` to focus on SHS in this how-to.

Reply:
Removed these parameters.